server/graphql/v2/mutation/AddFundsMutations.ts

```diff
@@ -16,6 +16,7 @@ export const addFundsMutation = {
     description: { type: new GraphQLNonNull(GraphQLString) },
     hostFeePercent: { type: new GraphQLNonNull(GraphQLInt) },
     platformFeePercent: { type: GraphQLInt, description: 'Can only be set if root' },
+    platformTipPercent: { type: GraphQLInt },
```

Comment:
I don't like this because it limits the tip to increments of 1% of the total value.
This also increases the complexity of the logic down below.
My suggestion is to always pass the `platformTip` as an absolute value in cents.

Reply:
@kewitz : I was wondering an alternative which is to make the `platformTipPercent` a float. The reason is that even if we remove the below logic, the `calcFee` will still need to be done within the (`getPlatformFeePercent`)[https://github.com/opencollective/opencollective-api/blob/059780c1a791adfb16380d3f9c7ce4d2e752fac1/server/lib/payments.js#L589] method. WDYT? 🤔

Reply:
You can just ignore the `getPlatformFeePercent` if you have the `platformTip`.
Why? Well, despite their name, basically they're too different things. Also, due to our business logic, we're deprecating platform fees in favor of tips and it is currently not possible to collect both on the same transaction.

Reply:
@kewitz : Thanks that makes sense. 👍🏽